src/main/java/org/terasology/navgraph/BaseRegion.java

```diff
@@ -20,30 +20,56 @@
 
 /**
  * @author synopia
+ * Abstract class which floors and regions derive from
  */
 public abstract class BaseRegion<N extends BaseRegion> {
     public int id;
     public Set<N> neighborRegions = new HashSet<N>();
     protected BitMap map;
     protected Floor floor;
 
+    /**
+     * Creates a BaseRegion
+     * @param id id of the base region
```

Comment:
Consider adding "the" between the two id's, so that it reads `@param id the id of the base region` to make it more readable - "id id" is a bit confusing